ledger/complete/wal/checkpoint_v6_test.go

```diff
@@ -167,3 +218,130 @@ func TestWriteAndReadCheckpointV6(t *testing.T) {
 		requireTriesEqual(t, tries, decoded)
 	})
 }
+
+func TestWriteAndReadCheckpointV6MultipleTries(t *testing.T) {
+	unittest.RunWithTempDir(t, func(dir string) {
+		tries := createMultipleRandomTries(t)
+		fileName := "checkpoint-multi-file"
+		logger := unittest.Logger()
+		require.NoErrorf(t, StoreCheckpointV6Concurrent(tries, dir, fileName, &logger), "fail to store checkpoint")
+		decoded, err := ReadCheckpointV6(dir, fileName, &logger)
+		require.NoErrorf(t, err, "fail to read checkpoint %v/%v", dir, fileName)
+		requireTriesEqual(t, tries, decoded)
```

Comment:
Maybe we can test checkpoint v6 roundtrip here:
- after `ReadCheckpointV6()` loads tries, call `StoreCheckpointV6Concurrent()` again to serialize tries to checkpoint files
- compare generated checkpoint files

Reply:
This has sort of tested by `TestCheckpointV6IsDeterminstic`. 

Since `ReadCheckpointV6` produces the same decoded tries as the original tries, then storing the tries again should be equivalent to storing the original tries again, which essentially to test if generating v6 is deterministic.